python/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "deltalake-python"
-version = "0.15.2"
+version = "0.16.0"
```

Comment:
Should we do version bump in a separate PR? Or rather should we come up with a strategy how we usually do this?

Reply:
Usually it's a separate one, but wanted to save time here in case this ended up in main this week : )